src/scenes/_Router/GalleryRedirect.tsx

```diff
@@ -8,9 +9,10 @@ type Props = {
 // 1) enables redirect without erroring
 // 2) wraps in Page component to prevent footer from flashing
 export default function GalleryRedirect({ to }: Props) {
-  return (
-    <Page>
-      <Redirect to={to} noThrow />
-    </Page>
-  );
+  const { replace } = useRouter();
+  useEffect(() => {
+    void replace(to);
+  }, [replace, to]);
```

Comment:
This is up for debate but I felt like the right behavior here is to replace the current route so they don't press the back button just to get redirected again.